projects/plugins/jetpack/extensions/blocks/mailchimp/test/fixtures/jetpack__mailchimp__deprecated-1.serialized.html

```diff
@@ -1,3 +1,3 @@
-<!-- wp:jetpack/mailchimp -->
```

Comment:
I've followed how failing tests were fixed in previous similar situations: https://github.com/Automattic/jetpack/pull/27443.